dev/ci/gitlab-ci-cases.yml

```diff
@@ -17,24 +17,24 @@
 # are referenced by name in the host configuration file's test matrix.
 
 # Template for experiment setup jobs
-.setup_template:
+.setup_experiment_template:
   extends: .base_config
-  stage: create_experiments
+  stage: setup_tests
   script:
     - |
-      export RUNTESTS="${RUNTESTS_DIR}"
-      "${GW_HOMEgfs}/dev/ci/scripts/utils/ci_utils.sh" create_experiment "${GW_HOMEgfs}/dev/ci/cases/pr/${caseName}.yaml" "${DATE}"
+      export RUNTESTS="${RUNTESTS_DIR}"                                                        # the dash delemeter is needed here as to not break pslot get parser
```

Comment:
What do you mean by `dash delemeter` in this comment?

Reply:
We disambiguated this comment with a push which is referring to pslot on line 26.

Reply:
OK, I see it belongs to the line above now.  Thanks.